cypress-io/cypress
# refactor: use Cypress design system icons in the SideBar

### User-facing changelog

None (it's a refactor)

### Additional details


### Steps to test

- Open the app 
- Hover the Sidebar items
- Check that the icons colors change in sync with the text


https://user-images.githubusercontent.com/5592465/179316728-b2ed68d5-d55d-4825-87b1-edfff8bae0ce.mp4



### How has the user experience changed?
Not

### PR Tasks


- N/A Have tests been added/updated?
- N/A Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- N/A Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- N/A Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
